.github/actions/label-and-move-released-issues-v1/dist/index.js

```diff
@@ -30722,21 +30743,21 @@ async function run(core, github) {
                     continue;
                 }
                 const { data: issue } = await octokit.rest.issues.get({
-                    repo,
-                    owner,
+                    repo: issueRepo,
+                    owner: issueOwner,
                     issue_number: issueNumber
                 });
                 if (issue.state !== 'closed') {
                     await octokit.rest.issues.update({
-                        repo,
-                        owner,
+                        repo: issueRepo,
+                        owner: issueOwner,
                         issue_number: issueNumber,
                         state: 'closed'
                     });
                 }
                 octokit.rest.issues.addLabels({
-                    repo,
-                    owner,
+                    repo: issueRepo,
```

Comment:
This is missing the required updates to cover the 2nd and 3rd bullet points from https://github.com/dequelabs/axe-api-team/issues/518. Particularly:

- The label format should change to include the name of the repo the action is running in (see the issue for details)
- The step that runs `issues.createLabel` if necessary (L54-72) needs to run for every repo you're touching issues in, not just the repo the action is running in. If you don't do this, the `addLabels` call here will still succeed, but it'll give the label an arbitrary default color instead of giving it the consistent fixed color that we set for same-repo cases.